### PR TITLE
Feature/update fta models

### PR DESCRIPTION
**Pull request by wesleymonte**

### CODE REVIEW

File: probes/src/main/java/cloud/fogbow/probes/core/probes/FogbowResourceAvailabilityProbe.java [hunk 1]
```diff
@@ -24,12 +23,14 @@
 
     public static final String THREAD_NAME = "Thread-Resource-Availability-Probe";
     private static final String PROBE_NAME = "resource_availability";
-    private static final String HELP = "Metric measures the level of failure to request a resource after your Order is Open.";
     private static final Logger LOGGER = LogManager
         .getLogger(FogbowResourceAvailabilityProbe.class);
 
     public FogbowResourceAvailabilityProbe(Integer timeSleep, String ftaAddress) {
         super(timeSleep, ftaAddress);
+        this.help = "Metric measures the level of failure to request a resource after your Order is Open.";
```
Comment (joivo) [inline probes/src/main/java/cloud/fogbow/probes/core/probes/FogbowResourceAvailabilityProbe.java]: Replace by `Measures the level of failure to request a resource after the Order is open.`

File: probes/src/main/java/cloud/fogbow/probes/core/probes/FogbowServiceLatencyProbe.java [hunk 2]
```diff
@@ -26,10 +25,12 @@
     private static final String COMPUTE_JSON_KEY = "COMPUTE";
     private static final String NETWORK_JSON_KEY = "NETWORK";
     private static final String VOLUME_JSON_KEY = "VOLUME";
-    private static final String HELP = "Latency is measured by the time that elapses between the order being opened until order are available.";
 
     public FogbowServiceLatencyProbe(Integer timeSleep, String ftaAddress) {
         super(timeSleep, ftaAddress);
+        this.help = "Latency is measured by the time that elapses between the order being opened until order are available.";
```
Comment (joivo) [inline probes/src/main/java/cloud/fogbow/probes/core/probes/FogbowServiceLatencyProbe.java]: Replace by `The time that elapses between the order being opened until the order is available.`

File: probes/src/main/java/cloud/fogbow/probes/core/probes/FogbowServiceReachabilityProbe.java [hunk 3]
```diff
@@ -46,6 +44,9 @@ public FogbowServiceReachabilityProbe(Integer timeSleep, String ftaAddress, Stri
         this.FNS_ENDPOINT = fnsEndpoint;
         this.MS_ENDPOINT = msEndpoint;
         this.services = Collections.unmodifiableMap(buildServices());
+        this.help = "Monitoring the availability of Fogbow services.";
```
Comment (joivo) [inline probes/src/main/java/cloud/fogbow/probes/core/probes/FogbowServiceReachabilityProbe.java]: Replace by `Returns 0 if the target service is not available or 1 if is.`

File: probes/src/main/java/cloud/fogbow/probes/core/probes/FogbowServiceSuccessRateProbe.java [hunk 4]
```diff
@@ -20,13 +19,15 @@
...
 
+    public static final String THREAD_NAME = "Thread-Service-Success-Rate-Probe";
     private static final String PROBE_NAME = "service_success_rate";
     private static final Logger LOGGER = LogManager.getLogger(FogbowServiceSuccessRateProbe.class);
-    private static final String HELP = "Measuring the success rate in requesting a resource.";
-    public static final String THREAD_NAME = "Thread-Service-Success-Rate-Probe";
 
     public FogbowServiceSuccessRateProbe(Integer timeSleep, String ftaAddress) {
         super(timeSleep, ftaAddress);
+        this.help = "Measuring the success rate in requesting a resource.";
```
Comment (joivo) [inline probes/src/main/java/cloud/fogbow/probes/core/probes/FogbowServiceSuccessRateProbe.java]: Remove the `measuring` word.

**Review by joivo — CHANGES_REQUESTED** [review]

File: probes/src/main/java/cloud/fogbow/probes/core/probes/FogbowResourceAvailabilityProbe.java (hunk 1 above)
Comment (wesleymonte) [reply probes/src/main/java/cloud/fogbow/probes/core/probes/FogbowResourceAvailabilityProbe.java]: Done.

File: probes/src/main/java/cloud/fogbow/probes/core/probes/FogbowServiceLatencyProbe.java (hunk 2 above)
Comment (wesleymonte) [reply probes/src/main/java/cloud/fogbow/probes/core/probes/FogbowServiceLatencyProbe.java]: Done.

File: probes/src/main/java/cloud/fogbow/probes/core/probes/FogbowServiceReachabilityProbe.java (hunk 3 above)
Comment (wesleymonte) [reply probes/src/main/java/cloud/fogbow/probes/core/probes/FogbowServiceReachabilityProbe.java]: Done.

File: probes/src/main/java/cloud/fogbow/probes/core/probes/FogbowServiceSuccessRateProbe.java (hunk 4 above)
Comment (wesleymonte) [reply probes/src/main/java/cloud/fogbow/probes/core/probes/FogbowServiceSuccessRateProbe.java]: Done.

**Review by joivo — APPROVED** [review]

**Pull request merged**